make_profiler/report_export.py

```diff
@@ -0,0 +1,92 @@
+#!/usr/bin/python3
+
+from datetime import datetime
+DATE_FORMAT = "%Y-%m-%dT%H:%M:%SZ"
+
+
+def escape_json_string(s):
+    s=str(s)
+    s = s.replace("\"", "")
+    s = s.replace("'","")
+    s = s.replace("\\","\\\\") # single \ to double \\
+    return s
+
+
+def export_report(performance,docs):
+    strOutputFile = 'report.json'
```

Comment:
It's better to choose one format for variables naming `camelCase` or `snake_case`. [Pep-8](https://peps.python.org/pep-0008/#function-and-variable-names) recommends to use snake case

Reply:
used snake_case